doc-content/enterprise-only/admin-and-config/proc-configuring-kie-server-persistence.adoc

```diff
@@ -0,0 +1,66 @@
+[id='proc-configuring-kie-server-persistence_{context}']
+
+= Configuring {KIE_SERVER} Persistence
+
+You can configure the {KIE_SERVER} persistence by passing hibernate or JPA parameters as system properties.
+
+To pass the hibernate or JPA parameters, use the following prefixes in your system properties:
+
+* `javax.persistence`
+* `hibernate`
+
+.Procedure
+
+. To configure {KIE_SERVER} persistence, complete any of the following tasks:
+
++
+--
+If you want to configure {KIE_SERVER} persistence using {EAP} configuration file, complete the following tasks:
```

Comment:
I'd suggest adding numbering or bullet to this point. I think bullet would be suitable.

Reply:
Updated. Added a bullet to this point.